src/components/Forms/RegisterForm.vue

```diff
@@ -253,7 +254,7 @@ export default {
 };
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
 @import 'settings';
```

Comment:
I had to remove scoped from this file's css because I needed to override the height attribute of the password strength meter badge.

Reply:
you can style child components using the `/deep/` selector: https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors

Reply:
😎 thanks @emuvente